homeassistant/components/sensor/const.py

```diff
@@ -484,6 +484,9 @@ class SensorStateClass(StrEnum):
     MEASUREMENT = "measurement"
     """The state represents a measurement in present time."""
 
+    MEASUREMENT_CIRCULAR = "measurement_circular"
```

Comment:
I would suggest naming it an angle.

```suggestion
    ANGLE = "angle"
```

Reply:
I called added measurement on purpose, so devs know that the same restrictions to measurement apply also to this one. Should we call it maybe `MEASUREMENT_ANGLE`?